NeuroID/src/reactNative/java/com/neuroid/tracker/extensions/NIDRNBuilder.kt

```diff
@@ -0,0 +1,62 @@
+package com.neuroid.tracker.extensions
+
+import android.app.Application
+import android.util.Log
+import androidx.annotation.VisibleForTesting
+import com.facebook.react.bridge.ReadableMap
+import com.neuroid.tracker.NeuroID
+
+class NIDRNBuilder( val application: Application? = null,
+                    val clientKey: String = "",
+                    private val rnOptions: ReadableMap? = null) {
+    fun build() {
+        val options = parseOptions(rnOptions)
+        // call NeuroID builder to initialize NeuroID with the options.
+        Log.d("NIDRNBuilder", "set options: $options")
+        NeuroID.Builder(application, clientKey,
+            options[RNConfigOptions.isAdvancedDevice] as Boolean,
+            options[RNConfigOptions.environment] as String).build()
```

Comment:
Could we move the setIsRN() method to here so we are consistent with iOS?

Reply:
yeah I can do that.

Reply:
updated

Reply:
on iOS it looks like setIsRN() is in both places. you want it both places like iOS?